node/src/commands/configure.js

```diff
@@ -16,14 +17,16 @@ const getQuestions = configuration => {
   }));
 };
 
+const removeEmptyAnswers = answers => _.pickBy(answers, i => i);
+
 const configure = async () => {
   const configuration = { ...emptyConfig, ...configManager.read() };
 
   const questions = getQuestions(configuration);
 
   const answers = await inquirer.prompt(questions);
 
-  configManager.write(answers);
+  configManager.write(removeEmptyAnswers(answers));
```

Comment:
This is due to how `inquirer` works. Expected behavior should be that if a person entered an empty string then he wanted to remove that config from disk so we should not save empty string.